chartpress.yaml

```diff
@@ -9,7 +9,7 @@ charts:
       hub:
         valuesPath: hub.image
         buildArgs:
-          JUPYTERHUB_VERSION: 1.0.0
+          JUPYTERHUB_VERSION: git+https://github.com/jupyterhub/jupyterhub@89b0c42
```

Comment:
Would a release tag be clearer than the hash here?

Reply:
100%! There is no JupyterHub release containing https://github.com/jupyterhub/jupyterhub/pull/2750 yet. I hope to merge this with a proper JupyterHub release tagged. I'm marking this as a TODO for the PR!